roles/setup_server/tasks/main.yaml

```diff
@@ -3,39 +3,48 @@
 - name: Get Package Facts
   ansible.builtin.package_facts:
 
-- name: Setup Repositories
-  community.general.rhsm_repository:
-    state: enabled
-    purge: true
-    name:
-      - "rhel-{{ ansible_distribution_major_version }}-for-x86_64-appstream-rpms"
-      - "rhel-{{ ansible_distribution_major_version }}-for-x86_64-baseos-rpms"
+- name: Test for ostree-based OS
+  ansible.builtin.shell:
+    cmd: "rpm-ostree status | grep Deployments"
+  register: ostree_check
```

Comment:
Let's namespace this and add a leading underscore since it's for internal bookkeeping, this way we won't accidentally clobber an existing user defined variable. Suggestion: _infra_osbuild_ostree_check

Reply:
Updated via https://github.com/redhat-cop/infra.osbuild/pull/60/commits/bb2ca613abe6cd3c01d8fe877706460e1d8c6316